tests/orion/api/test_block_schemas.py

```diff
@@ -160,6 +160,17 @@ async def test_create_block_schema_for_system_block_type_fails(
             == "Block schemas for protected block types cannot be created."
         )
 
+    async def test_create_block_schema_for_system_block_type_does_not_fail_for_older_clients(
+        self, system_block_type, client_with_unprotected_block_api
+    ):
+        response = await client_with_unprotected_block_api.post(
+            "/block_schemas/",
+            json=BlockSchemaCreate(fields={}, block_type_id=system_block_type.id).dict(
+                json_compatible=True
+            ),
+        )
+        assert response.status_code != 403
```

Comment:
```suggestion
        assert response.status_code != 201
```

Reply:
Can you use the `fastapi.status` enums?

Reply:
I do think this is mildly out of scope since I didn't want to go and change every assertion in the block schema test suite but I went ahead and did this anyway

Reply:
👍 thanks for taking it on anyway. Didn't realize the whole file was out of date.